forms-flow-web/src/components/Form/Create.js

```diff
@@ -337,6 +333,39 @@ const Create = React.memo(() => {
             </div>
           </div>
           <div className="col-lg-4 col-md-4 col-sm-4">
+            <div id="form-group-supportedidp" className="form-group">
+              <label htmlFor="supportedidp" className="control-label field-required">
+                <Translation>{(t) => t("Supported IDPs (comma separated)")}</Translation>
+                {addingTenantKeyInformation("supportedidp")}
+              </label>
+              <div className="input-group mb-2">
+                {MULTITENANCY_ENABLED && tenantKey ? (
+                  <div className="input-group-prepend">
+                    <div
+                      className="input-group-text"
+                      style={{ maxWidth: "150px" }}
+                    >
+                      <span className="text-truncate">{tenantKey}</span>
+                    </div>
+                  </div>
+                ) : (
+                  ""
+                )}
+                <input
+                  type="text"
+                  className="form-control"
+                  id="supportedidp"
+                  placeholder={t("Supported IDPs (comma separated)")}
```

Comment:
I would make an example here like `idir,bceid`

Reply:
Just update it e.g. idir or idir,bceid